src/current/v23.1/monitoring-and-alerting.md

```diff
@@ -189,6 +189,685 @@ If you rely on external tools for storing and visualizing your cluster's time-se
 
 When storage of time-series metrics is disabled, the DB Console Metrics dashboards in the DB Console are still available, but their visualizations are blank. This is because the dashboards rely on data that is no longer available.
 
+### Critical nodes endpoint
+
+The critical nodes status endpoint is used to:
+
+- Check if any of your nodes are in a critical state.  A node is _critical_ if that node becoming unreachable would cause [replicas to become unavailable](ui-cluster-overview-page.html#replication-status).
```

Comment:
```suggestion
- Check if any of your nodes is in a critical state. A node is _critical_ if that node becoming unreachable would cause [replicas to become unavailable](ui-cluster-overview-page.html#replication-status).
```